spec/features/dams_object_spec.rb

```diff
@@ -1495,14 +1500,26 @@
     expect(page).to have_content('Generic Component Title 2')
     expect(page).to_not have_link('', href:"/object/#{@obj.id}/_2_1.mp4/download?access=curator")
   end
+
+  scenario 'user should see an embed link' do
+    Capybara.javascript_driver = :poltergeist
+    Capybara.current_driver = Capybara.javascript_driver
+    sign_in_developer
+    visit dams_object_path @obj.pid
+    expect(page).to have_css("a", :text => "Embed")
+    click_button 'component-pager-forward'
+    expect(page).to have_content('Generic Component Title 2')
+    expect(page).to have_css("a", :text => "Embed")
+  end
+
```

Comment:
Is it possible to add a test for the component embed UI like `embed/#{id}/2`?

Reply:
Test added.